### ISSUE TITLE
Maru os on lineage 15.1 (android 8.1)

### ISSUE THREAD

**Issue by dematteisF** [maruos/maruos#94]
Hi,

Not really an issue, but a question - is there any way to have MaruOS on lineage? I have 15.1 version (Android 8.1 ) on my phone (nexus 5) now, and it really fantastic rom.

Thanks!

**Comment by pdsouza** [comment]
Thanks for your question.

We are based on vanilla AOSP up through our current stable release. For our next major release, we are in the process of moving to Lineage 14.1 in a move to support more diverse hardware rather than just Nexus devices. I have managed to get some dev builds running on the Nexus 5 already. Some polish is needed before releasing builds but we should get there before too long.

Also, one of our community members said he would try to get things running on 15.1 for some of the newer devices. It's a hit or miss on upgrading Android versions...sometimes it's not too difficult like when we moved from Android 5 to 6, but sometimes it can be a pain like with 6 to 7 if some internal APIs we use change. We'll see. 14.1 works today though, so we'll move to that for now.

**tl;dr** Next Maru major release will be based on Lineage 14.1. We will experiment with moving to 15.1 afterwards.

**Comment by dematteisF** [comment]
Fantastic! Thank you!

De matteis Francesco

Il gio 7 giu 2018, 05:10 pdsouza <notifications@github.com> ha scritto:

**Comment by pdsouza** [comment]
0.6 is now available for all and brings Maru to Lineage OS 15.1 / Android 8.1.0 Oreo!
